pyproject.toml

```diff
@@ -44,6 +44,7 @@ opencv-python = { version = ">=4.5.5.64", optional = true }
 opencv-python-headless = ">=4.5.5.64"
 requests = { version = ">=2.26.0,<=2.31.0", optional = true }
 tqdm = { version = ">=4.62.3,<=4.66.1", optional = true }
+types-pillow = "^10.2.0.20240206"
```

Comment:
@onuralpszr I'm curious about your opinion here.

Reply:
I added this and `poetry.lock` by mistake. This commit should only have touched `annotators/base.py`.

Shall I rever that?

Reply:
Hi @LinasKo, my main question is how will you run `annotated = Image.fromarray(annotated)` without Pillow?

Reply:
@LinasKo I'm also updating my previous comment about docs. Please give me few minutes.

Reply:
Oh, that's right, you don't have Pillow as a dependency at all? That's curious.

Seems like me having it installed when calling it was enough for some reason.

Yes, it needs to be added here.

Reply:
Feel free to add it. ;)

Reply:
@LinasKo  @SkalskiP  let's remove that and If we REALLY needed it. we can into dev package but for main package %100 no let's remove from there